go/chat/storage/storage.go

```diff
@@ -642,6 +644,31 @@ func (s *Storage) applyExpunge(ctx context.Context, convID chat1.ConversationID,
 	return &expunge, nil
 }
 
+func (s *Storage) ClearBelow(ctx context.Context, convID chat1.ConversationID, uid gregor1.UID,
+	msgID chat1.MessageID) (err Error) {
```

Comment:
`ClearBefore` would be a clearer name since later messages appear 'below' in the UIs.

Comment whether `msgID` is exclusive or inclusive. If it's exclusive calling `upto` helps communicate that (I think).